patches/minecraft/net/minecraft/entity/SharedMonsterAttributes.java.patch

```diff
@@ -0,0 +1,11 @@
+--- ../src-base/minecraft/net/minecraft/entity/SharedMonsterAttributes.java
++++ ../src-work/minecraft/net/minecraft/entity/SharedMonsterAttributes.java
+@@ -16,7 +16,7 @@
+ public class SharedMonsterAttributes
+ {
+     private static final Logger field_151476_f = LogManager.getLogger();
+-    public static final IAttribute field_111267_a = (new RangedAttribute((IAttribute)null, "generic.maxHealth", 20.0D, 0.0D, 1024.0D)).func_111117_a("Max Health").func_111112_a(true);
++    public static final IAttribute field_111267_a = (new RangedAttribute((IAttribute)null, "generic.maxHealth", 20.0D, 1.0D, 1024.0D)).func_111117_a("Max Health").func_111112_a(true);
```

Comment:
I tried Double.MIN_VALUE and Double.MIN_NORMAL. I saw a death screen. I may have mistaken it for a client desync bug I discovered later, though. At some point this pull request thread was starting to make my blood boil, so I had to take a break.